src/javascript.js

```diff
@@ -4,5 +4,9 @@
 // selecte element :
 const numberinput = document.getElementById("number-input");
 const favouriteBtn = document.getElementById("favourite-Btn");
+// const header1 = document.getElementsByClassName("header")[0];
 
-// your function:
+favouriteBtn.addEventListener("click", () => {
+
+  console.log(`When you click no favourite it will count numbers of clicks ${numberInput}`);
+})
```

Comment:
@sardarMrashed good that you tried your best but.
you may misunderstood the task and its as follows =>

- when favouriteBtn is clicked i want to console log the value that is inside the number input
-imagine you put 1 in the number input then click on favouriteBtn ,  then in the console should appear 1 and so on.